pkgs/epnix/support/sscan/default.nix

```diff
@@ -1,29 +1,27 @@
 {
-  lib,
   epnixLib,
   mkEpicsPackage,
   fetchFromGitHub,
-  fetchpatch,
   epnix,
   local_config_site ? {},
   local_release ? {},
 }:
 mkEpicsPackage rec {
   pname = "sscan";
-  version = "2-11-5";
+  version = "2-11-6";
   varname = "SSCAN";
 
-  inherit local_config_site local_release;
-
-  buildInputs = with epnix.support; [seq];
-
   src = fetchFromGitHub {
     owner = "epics-modules";
-    repo = "sscan";
+    repo = pname;
```

Comment:
The repo name changed from `sscan` to `pname`?

I don't get it, the repo still looks it's called `sscan`: <https://github.com/epics-modules/sscan>
Isn't it?

Reply:
OK! That's nice!